cmd_actuators_mux/README.md

```diff
@@ -0,0 +1,14 @@
+# Command Actuator Multiplexer.
```

Comment:
@patapufate same as above, it is good to separate package from node.

Reply:
Done!